### ISSUE TITLE
SHMEM_TEAM_SPLIT_STRIDED expects global stride instead of relative stride?

### ISSUE THREAD

**Issue by zhongchen530** [Sandia-OpenSHMEM/SOS#1151]
The function SHMEM_TEAM_SPLIT_STRIDED takes in a parent team, start, size, and stride argument to produce a new team. I would expect in this case the stride to be relative to the parent team. However, what I observed was that it expects the stride to be relative to the SHMEM_TEAM_WORLD instead.

For instance if number of PEs is 4, meaning SHMEM_TEAM_WORLD : {0,1,2,3,4,5}.

shmem_team_split_strided(SHMEM_TEAM_WORLD, 0, 2, 3, NULL, 0, &new_team);

would result in 

new_team : {0,2,4}

If I again call 

shmem_team_split_strided(new_team, 0, 2, 2, NULL, 0, &another_team);

this would result in 

another_team : {0, 2} 

The stride is still relative to the global PE but not the parent team new_team in this case.

Is this an intended behavior?

**Comment by davidozog** [comment]
@zhongchen530 - Oops!  I had to write all that to see that maybe SOS does _not_ behave how I described, and you found an issue.  Will investigate..

**Comment by zhongchen530** [comment]
> @zhongchen530 - Oops! I had to write all that to see that maybe SOS does _not_ behave how I described, and you found an issue. Will investigate..

Yes, your explanation above is what I initially expected, but it doesn't behave that way. Instead, ```another_team``` is observed to be ```{0,2}``` instead.

**Comment by davidozog** [comment]
Closed by #1153
